src/components/slider/slider.scss

```diff
@@ -0,0 +1,125 @@
+@import 'default-theme';
+
+$slider-size: 20px !default;
```

Comment:
Maybe rename rather than just adding a comment. Would `$slider-height` be more accurate?